README.md

```diff
@@ -36,17 +36,30 @@ You can deploy this app to the Fly hosting service using the CLI. Sign up at fly
 
 ### Backends
 
-[Backends](https://github.com/superfly/cdn/tree/master/src/backends) are origin services you can route requests to. The project includes a backend type [any HTTP service](https://github.com/superfly/cdn/blob/master/src/backends/origin.ts), and more specialized types for proxying to third party services.
+[Backends](https://github.com/superfly/cdn/tree/master/src/backends) are origin services you can route requests to. The project includes a backend type [any HTTP service](https://github.com/superfly/cdn/blob/master/src/backends/origin.ts), and more specialized types for proxying to third party services. 
```

Comment:
This sentence is missing a word, maybe should be "includes a backend type. Also that trailing space can go buhbye.